src/components/link/LinkBase.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+
+export interface LinkBaseProps {
+    children: React.ReactNode;
```

Comment:
I just noticed, I think we should start using React.ReactChild for children. I havent tried it though.

This is the official types:
```
   type ReactText = string | number;
    type ReactChild = ReactElement | ReactText;

    interface ReactNodeArray extends Array<ReactNode> {}
    type ReactFragment = {} | ReactNodeArray;
    type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```